static/src/javascripts/projects/commercial/modules/third-party-tags/plista.js

```diff
@@ -3,13 +3,16 @@ import fastdom from 'fastdom';
 import $ from 'lib/$';
 import config from 'lib/config';
 import detect from 'lib/detect';
-import template from 'lodash/utilities/template';
 import trackAdRender from 'commercial/modules/dfp/track-ad-render';
 import { commercialFeatures } from 'commercial/modules/commercial-features';
-import plistaStr from 'raw-loader!commercial/views/plista.html';
 import { loadScript } from 'lib/load-script';
 
-const plistaTpl = template(plistaStr);
+const plistaTpl = ({ widgetName }) => `
+    <div class="PLISTA" data-ob-template="guardian">
+        <div data-display="plista_widget_${widgetName}"></div>
+    </div>
+`;
+
```

Comment:
I've spent a fair while failing to get raw-loader to work with Jest. I'm very tempted to follow your lead and inline the template.